src/test/java/study/racing/domain/CarTest.java

```diff
@@ -38,4 +40,24 @@
         car.moveTheCar(input);
         assertThat(car.getDistance().getMoveDistance()).isEqualTo(expected);
     }
+
+    @DisplayName("차객체 생성시 이름주입 테스트")
+    @ParameterizedTest
+    @CsvSource(value = {"test1,test1","test!,test!","12345,12345","가나다라마,가나다라마"})
+    void 차이름입력테스트(String input, String expected) {
+        Car car = Car.createCar(input);
+        assertThat(car.getName().getCarName()).isEqualTo(expected);
+    }
+
+    @DisplayName("차객체 이동거리 최대값 판별 테스트")
+    @ParameterizedTest
+    @CsvSource(value = {"test1,6,0","test1,5,0"})
+    void 차이동거리최대값테스트(String input, int move, int target) {
+        Car car = Car.createCar(input);
+        car.moveTheCar(move);
+
+        int max = car.maxDistance(car.getDistance().getMoveDistance());
+
+        assertThat(max).isGreaterThan(target);
```

Comment:
```suggestion
        int max = car.maxDistance(0);

        assertThat(max).isEquals(1);
```
테스트가 의도하는 바를 명확하게 이해하기가 어렵네요.

Reply:
TDD를 통해서 Car 클래스에 메소드를 만드는 식으로 접근하면서 생성된 테스트코드네요.

랜덤값을 주입하지 않고 4보다 큰 값을 주입해서 이동이 발생하게 테스트 값을 유도했을때

정상적으로 수행이 된다면 비교값인 0이 아닌 1이 리턴받게 될 것이고 
이것을 수비교 형식으로 표현해보려고 했습니다.